nemo/collections/asr/modules/audio_preprocessing.py

```diff
@@ -307,3 +467,75 @@ def forward(self, input_spec):
         augmented_spec = self.spec_cutout(input_spec)
         augmented_spec = self.spec_augment(augmented_spec)
         return augmented_spec
+
+
+class CropOrPadSpectrogramAugmentation(NeuralModule):
```

Comment:
let's add @experimental until there are basic unittests

Reply:
Added basic test for this module